code/__HELPERS/text.dm

```diff
@@ -269,7 +269,9 @@
 	return ""
 
 //Returns a string with reserved characters and spaces before the first word and after the last word removed.
-/proc/trim(text)
+/proc/trim(text, max_length)
+	if(max_length)
+		text = copytext_char(text, 1, max_length)
```

Comment:
IMO, this should not be included as part of this function, it should stay simple, and if someone needs to set a max length, they can do that.